content/zh/docs/tasks/administer-cluster/configure-upgrade-etcd.md

```diff
@@ -0,0 +1,451 @@
+---
+reviewers:
+- mml
+- wojtek-t
+title: 为 Kubernetes 运行 etcd 集群
+content_template: templates/task
+---
+<!--
+---
+reviewers:
+- mml
+- wojtek-t
+title: Operating etcd clusters for Kubernetes
+content_template: templates/task
+---
+-->
+
+{{% capture overview %}}
+
+{{< glossary_definition term_id="etcd" length="all" prepend="etcd is a ">}}
```

Comment:
BTW, `"etcd is a "` is untranslated.

Reply:
@sftim, how to translate that sentence, that seems like a half sentence.

Reply:
+1

Reply:
I have modified it. I don't know if it is suitable?

Reply:
This might work:
```markdown
{{< glossary_definition term_id="etcd" length="all">}}
```

I'm afraid I don't speak Chinese well enough to comment on how the page comes out with that shortcode.

Reply:
@sftim Thank you very much for your suggestion, I will think about how to translate.